sdk/enter.go

```diff
@@ -80,6 +80,62 @@ func simpleChrootHelper(args []string) error {
 		return fmt.Errorf("Mounting %q failed: %v", newRepoRoot, err)
 	}
 
+	homedir := os.Getenv("HOME")
+	if homedir == "" {
+		return fmt.Errorf("HOME environment variable is not set.")
+	}
+
+	// mount the directory containing the ssh-agent socket in order
+	// to support a private repos during repo sync
+	sshAuthSock := os.Getenv("SSH_AUTH_SOCK")
```

Comment:
Although considering redefining `$SSH_AUTH_SOCK` is a behavior change from `enter_chroot.sh` that could come later. As functionality moves into Go code matching the old behavior is sufficient for now.